backend/de.metas.contracts/src/main/java/de/metas/contacts/invoice/interim/process/C_Flatrate_Term_CreateInterimContract.java

```diff
@@ -56,11 +65,21 @@ public ProcessPreconditionsResolution checkPreconditionsApplicable(final @NonNul
 		{
 			return ProcessPreconditionsResolution.rejectBecauseNotSingleSelection();
 		}
-		final I_C_OrderLine salesOrder = context.getSelectedModel(I_C_OrderLine.class);
-		if (!salesOrder.isProcessed())
+		final I_C_OrderLine orderLine = InterfaceWrapperHelper.load(context.getSingleSelectedRecordId(), I_C_OrderLine.class);
+		if (!orderLine.isProcessed())
 		{
 			return ProcessPreconditionsResolution.rejectWithInternalReason("only processed order lines are allowed");
 		}
+		final I_C_Order order = InterfaceWrapperHelper.load(orderLine.getC_Order_ID(), I_C_Order.class);
```

Comment:
Please, use de.metas.order.impl.AbstractOrderDAO#getById(de.metas.order.OrderId)

Reply:
and done.